challenge.rb

```diff
@@ -1,26 +1,27 @@
 
-def capitalize_each_string(input)
+def capitalize_each_string(*animal)
   #implement your solution here
+  animal.each { |a| puts a.capitalize }
```

Comment:
Hey Nicolas,
Thanks for taking a look at this.
I just noticed that the code at the bottom of the file had the array change to all caps.  Is this what you are referring to?  I believe using the "collect" or "map" method should do this...right?  I used the splat operator because I was testing the method using an array and allowing an undefined number of parameters.

I tested the method in Interactive ruby and the code seemed to pass...not sure what else the issue might be.